data/fro/Eusebi_1963_ChevOgierDanemarche/metadata.txt

```diff
@@ -0,0 +1,76 @@
+publicationType : "book"
+publicationTitle : "La chevalerie d’Ogier de Danemarche, canzone di gesta"
+publicationDate : "1963"
+publicationPlaces:
+    - publicationPlace: "Milano"
+publicationSeries : "Testi e documenti di letteratura moderna"
+publicationNum: "6"
+publicationStemmaNum: ""
+publicationAuthors:
+    - publicationAuthor: "Eusebi, Mario"
+publicationPage : ""
+publicationLink : ""
+workTitle : "Chevalerie d’Ogier de Danemarche"
+workViaf : "189404871"
+workOrigDate : "1192-1200"
+workOrigPlace : "Paris"
+workAuthors:
+    - workAuthor: "Raimbert de Paris"
+      workAuthorViaf: "188921350"
+workGenre : "chanson de geste"
+workLangCode : "fro"
+stemmaType : "reconstructed"
+drawnStemma : "true"
+contam : "no"
+extraStemmContam : "no"
+rootType : "original"
+completeWits : "included"
+sourceText : "NA"
+derivatives : "NA"
+contributors:
+    - contributor: "Jean-Baptiste Camps"
+      contributorORCID: "0000-0003-0385-7037"
+note : "All known witnesses except the fragment Saint-Étienne, Médiathèque Marc-Ferro,  Cote: 5,  Ancienne cote: D3 502"
+wits:
+    - witSigla: "D"
+      witSignature: "Durham, University Library, Cosin, V. II. 17"
```

Comment:
Maybe I could actually improve the TEI conversion, but right now we would need to remove the comma after "Cosin" so that it parses settlement-repository-idno correctly.

Reply:
Seems like a good rule, I've corrected on my side.